docs/content/latest/explore/multi-region-deployments/asynchronous-replication-ycql.md

```diff
@@ -95,36 +95,48 @@ Waiting for cluster to be ready.
 
 In the default `yugabyte` database, create the database table `users` on the "Data Center - East" cluster.
 
-Open `ysqlsh` specifying the host IP address of `127.0.0.1`.
+Open `ycqlsh` specifying the host IP address of `127.0.0.1`.
 
 ```sh
-$ ./bin/ysqlsh -h 127.0.0.1
+$ ./bin/ycqlsh 127.0.0.1
+```
+Create a keyspace by running the following statement.
+
+```sql
+ycqlsh> CREATE KEYSPACE customers;
+```
+
+```sql
+ycqlsh> USE customers;
 ```
 
 Run the following `CREATE TABLE` statement.
 
-```plpgsql
-CREATE TABLE users (
-    email varchar(35) PRIMARY KEY,
-    username varchar(20)
-    );
+``` sql
+CREATE TABLE users ( email varchar PRIMARY KEY, username varchar );
+
 ```
 
 Now create the identical database table on cluster B.
 
-Open `ysqlsh` for "Data Center - West" by specifying the host IP address of `127.0.0.2`.
+Open `ycqlsh` for "Data Center - West" by specifying the host IP address of `127.0.0.2`.
 
 ```sh
-$ ./bin/ysqlsh -h 127.0.0.2
+$ ./bin/ycqlsh 127.0.0.2
+```
+Create a keyspace by running the following statement.
+
+```sql
+ycqlsh> CREATE KEYSPACE customers;
 ```
 
+```sql
+ycqlsh> USE customers;
+```
 Run the following `CREATE TABLE` statement.
 
 ```plpgsql
-CREATE TABLE users (
-    email varchar(35) PRIMARY KEY,
-    username varchar(20)
-    );
+CREATE TABLE users ( email varchar PRIMARY KEY, username varchar );
```

Comment:
we could combine all three code blocks into one for this step, since it's just repetition of the datacenter-east creation...